pkg/operators/v1alpha1/catalogsource_types.go

```diff
@@ -2,12 +2,11 @@ package v1alpha1
 
 import (
 	"fmt"
-	corev1 "k8s.io/api/core/v1"
-	"time"
-
 	"github.com/sirupsen/logrus"
+	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
+	"time"
```

Comment:
Interesting - I'm surprised the format check didn't complain about this but I guess gofmt might no care about import ordering/chunking/etc.?

Reply:
It doesn't have an opinion about std vs. external module chunking, just sorts within chunks. goimports does care i think.